frontend/packages/gitops-plugin/src/components/GitOpsListPage.tsx

```diff
@@ -43,41 +41,23 @@ const GitOpsListPage: React.FC = () => {
     };
   }, [baseURL, namespaces, nsError, nsLoaded, t]);
 
-  const [consoleLinks] = useK8sWatchResource<K8sResourceKind[]>({
-    isList: true,
-    kind: referenceForModel(ConsoleLinkModel),
-    optional: true,
-  });
-  const argocdLink = _.find(
-    consoleLinks,
-    (link: K8sResourceKind) =>
-      link.metadata?.name === 'argocd' && link.spec?.location === 'ApplicationMenu',
-  );
-
   return (
     <>
       <Helmet>
         <title>{t('gitops-plugin~Environments')}</title>
       </Helmet>
-      <PageHeading
-        title={t('gitops-plugin~Environments')}
-        badge={
-          <Split className="odc-gitops-list-page-heading" hasGutter>
-            <SplitItem>
-              {argocdLink && (
-                <ExternalLink href={argocdLink.spec.href} text={t('gitops-plugin~Argo CD')} />
-              )}
-            </SplitItem>
-            <SplitItem>
-              <DevPreviewBadge />
-            </SplitItem>
-          </Split>
-        }
-      />
+      <PageHeading title={t('gitops-plugin~Environments')} badge={<DevPreviewBadge />} />
       {!appGroups && !emptyStateMsg ? (
         <LoadingBox />
       ) : (
-        <GitOpsList appGroups={appGroups} emptyStateMsg={emptyStateMsg} argocdLink={argocdLink} />
+        <>
+          <PageHeading className="co-catalog-page__description">
+            {t(
+              `gitops-plugin~Select an application to view the environment it&apos;s deployed in.`,
```

Comment:
```suggestion
              "gitops-plugin~Select an application to view the environment it's deployed in.",
```
Also you'll need to add this to the translation file & run `yarn i18n`